configs/base_slurm.yml

```diff
@@ -0,0 +1,12 @@
+# submitit
+slurm:
+  submit: false
+  name: "VidFeats"
+  log_folder_path: "./output"
+  num_arrays: 2
+  max_arrays: 10 # Max num arrays to run
+  mem_gb: 50 # Memory per task in GB
+  cpus_per_task: 10 # CPU cores to use per task
+  timeout_hrs: 1 # Job length in hours
+  partition: "gpu"
```

Comment:
i think, `partition` must be user-specified because it varies across clusters. should we remove it from the default config, add this argument to the readme, and add an assertion that it is not `None` (specified)?